io.gitlab.leesonwai.Sums.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id" : "io.gitlab.leesonwai.Sums",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "40",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "sums",
+  "finish-args" : [
+    "--socket=fallback-x11",
+    "--socket=wayland"
+  ],
+  "cleanup" : [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules" : [
+    {
+      "name" : "libadwaita",
+      "buildsystem" : "meson",
+      "config-opts" : [
+        "-Dexamples=false",
+        "-Dintrospection=disabled",
+        "-Dtests=false",
+        "-Dvapi=false"
+      ],
+      "sources" : [
+        {
+          "type" : "git",
+          "url" : "https://gitlab.gnome.org/exalm/libadwaita.git",
+          "branch" : "main"
```

Comment:
you need to specify a commit as a branch moves.

Reply:
Done.